extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -424,28 +457,25 @@ export class AmpConsent extends AMP.BaseElement {
   }
 
   /**
-   * Handler User action
+   * Checks if we are in a valid state to handle user actions.
+   * @return {boolean}
+   */
+  isReadyToHandleAction_() {
+    if (!this.consentStateManager_) {
+      dev().error(TAG, 'No consent state manager');
+      return false;
+    }
+    return this.isPromptUIOn_;
```

Comment:
nit: `isPromptUiOn` Sorry, missed this earlier.

Reply:
Yea it's weird, b/c in our config we have `promptUI`, `promptUISrc`, and `postPromptUI`... should get to changing this (maybe a fixit week item 👀)